docs/source/howto/installation.rst

```diff
@@ -487,68 +487,98 @@ Updating from 0.x.* to 1.*
+The panels below provide instructions for storage backends provided by ``aiida-core``.
+To determine what storage backend a profile uses, call ``verdi profile show``.
 
+.. tab-set::
 
-.. todo::
+    .. tab-item:: psql_dos
 
-    .. _how-to:installation:backup:repository:
+        The ``psql_dos`` storage backend is the default backend for AiiDA.
+        It stores its data in a PostgreSQL database and a file repository on the local filesystem.
+        To fully backup the data stored for a profile, you should backup the associated database and file repository.
 
-    title: Repository backup
+        **PostgreSQL database**
 
+        To export the entire database, we recommend to use the `pg_dump <https://www.postgresql.org/docs/12/app-pgdump.html>`_ utility:
 
-.. _how-to:installation:backup:postgresql:
+        .. code-block:: console
 
-Database backup
----------------
+            pg_dump -h <database_hostname> -p <database_port> -d <database_name> -W > aiida_backup.psql
 
-PostgreSQL typically spreads database information over multiple files that, if backed up directly, are not guaranteed to restore the database.
-We therefore strongly recommend to periodically dump the database contents to a file (which you can then back up using your method of choice).
+        The ``-W`` flag will ensure to prompt for the database password.
+        The parameters between brackets should be replaced with the values that have been configured for the profile.
+        You can retrieve these from the ``storage.config`` returned by the ``verdi profile show`` command.
 
-A few useful pointers:
+        .. tip::
 
-* In order to avoid having to enter your database password each time you use the script, you can create a file ``.pgpass`` in your home directory containing your database credentials, as described `in the PostgreSQL documentation <https://www.postgresql.org/docs/12/libpq-pgpass.html>`_.
+            In order to avoid having to enter your database password each time you use the script, you can create a file ``.pgpass`` in your home directory containing your database credentials, as described `in the PostgreSQL documentation <https://www.postgresql.org/docs/12/libpq-pgpass.html>`_.
 
-* In order to dump your database, use the `pg_dump utility from PostgreSQL <https://www.postgresql.org/docs/12/app-pgdump.html>`_. You can use as a starting example a bash script similar to :download:`this file <include/backup_postgresql.sh>`.
+        **File repository**
+
+        The file repository is a directory on the local file system.
+        The most efficient way to create a backup is to use the `rsync <https://en.wikipedia.org/wiki/Rsync>`_ utility.
+        The path of the repository is shown in the ``storage.config.repository_uri`` key returned by the ``verdi profile show`` command.
+        To create a backup, simply run:
+
+        .. code-block:: console
+
+            rsync -arvz <storage.config.repository_uri> /some/path/aiida_backup
```

Comment:
Should they not always run `verdi storage maintain` first. Otherwise you are actually rsyncing all the loose files

Reply:
It is not strictly required of course, but I added a tip block suggesting they run it in order to optimize the backup speed.